grill/runner.go

```diff
@@ -115,15 +115,20 @@ func (t *Test) Run(ctx TestContext) error {
 		return fmt.Errorf("couldn't run command: %s", err)
 	}
 	if err = cmd.Wait(); err != nil {
-		exitErr, ok := err.(*exec.ExitError)
-		if ok {
+		if exitErr, ok := err.(*exec.ExitError); ok {
 			buf.Write(exitErr.Stderr)
 			status := exitErr.Sys()
 			if s, ok := status.(syscall.WaitStatus); ok {
 				fmt.Fprintf(buf, "[%d]", s.ExitStatus())
 			}
 			err = nil
 		}
+		// TODO else?
```

Comment:
Docs say other errors can be returned, but this is pretty fatal, IMO panic.

Reply:
Done